core/src/main/java/org/apache/accumulo/core/client/impl/MasterClient.java

```diff
@@ -183,4 +193,13 @@ public static void executeVoid(ClientContext context, ClientExec<MasterClientSer
       throw new AssertionError(e);
     }
   }
+
+  public static void executeVoidAdmin(ClientContext context, ClientExec<MasterClientService.Client> exec) throws AccumuloException, AccumuloSecurityException {
```

Comment:
Maybe call this `executeVoidWithTimeout`? Again, passing timeout here, and then down through, would be helpful. The timeout used for `Admin.java` should be a constant in that class.

Reply:
Please check my comment  above.